en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/getting-started/quick-start-guide/quick-start-guide-kubernetes.md

```diff
@@ -21,8 +21,8 @@ Let's get started...
 
 ### Step 1 - Setup Microgateway and K8s API Operator in Kubernetes
 
-1.  Download the WSO2 API Microgateway v4.0.0-m3 from
-    [github release page's](https://github.com/wso2/product-microgateway/releases/tag/v4.0.0-m3) assets and extract them
+1.  Download the WSO2 API Microgateway v4.0.0-m10 from
+    [github release page's](https://github.com/wso2/product-microgateway/releases/tag/v4.0.0-m10) assets and extract them
```

Comment:
Let's use the alpha of MG. Alpha will be released today.

Reply:
updated with https://github.com/wso2/docs-apim/pull/3491/files/c69498fda8ea36ec12416429665bc896c8c11f3f..8a12993936ffb869b0d299dc1ee16107b7a95d7f